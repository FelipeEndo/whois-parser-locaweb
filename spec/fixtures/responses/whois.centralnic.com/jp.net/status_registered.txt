Domain Name: NTT.JP.NET
Registry Domain ID: D195732259-CNIC
Registrar WHOIS Server: whois.1api.net
Registrar URL: http://www.1api.net
Updated Date: 2020-09-04T04:22:52.0Z
Creation Date: 2020-08-06T08:41:29.0Z
Registry Expiry Date: 2022-08-06T23:59:59.0Z
Registrar: 1API GmbH
Registrar IANA ID: 1387
Domain Status: ok https://icann.org/epp#ok
Registrant Email: https://whois.centralnic.com/contact/ntt.jp.net/registrant
Admin Email: https://whois.centralnic.com/contact/ntt.jp.net/admin
Tech Email: https://whois.centralnic.com/contact/ntt.jp.net/tech
Name Server: F1G1NS1.DNSPOD.NET
Name Server: F1G1NS2.DNSPOD.NET
DNSSEC: unsigned
Billing Email: https://whois.centralnic.com/contact/ntt.jp.net/billing
Registrar Abuse Contact Email: abuse@1api.net
Registrar Abuse Contact Phone: +49.68949396850
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-12-15T16:49:27.0Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

>>> IMPORTANT INFORMATION ABOUT THE DEPLOYMENT OF RDAP: please visit
https://www.centralnic.com/support/rdap <<<

The Whois and RDAP services are provided by CentralNic, and contain
information pertaining to Internet domain names registered by our
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com)

Access to the Whois and RDAP services is rate limited. For more
information, visit https://registrar-console.centralnic.com/pub/whois_guidance.
