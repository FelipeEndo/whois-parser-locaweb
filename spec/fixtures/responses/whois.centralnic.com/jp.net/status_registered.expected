#disclaimer
  %s == "The Whois and RDAP services are provided by CentralNic, and contain information pertaining to Internet domain names registered by our our customers. By using this service you are agreeing (1) not to use any information presented here for any purpose other than determining ownership of domain names, (2) not to store or reproduce this data in any way, (3) not to use any high-volume, automated, electronic processes to obtain data from this service. Abuse of this service is monitored and actions in contravention of these terms will result in being permanently blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com) Access to the Whois and RDAP services is rate limited. For more information, visit https://registrar-console.centralnic.com/pub/whois_guidance."



#domain
  %s == "ntt.jp.net"

#domain_id
  %s == "D195732259-CNIC"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2020-08-06 08:41:29 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2020-09-04 04:22:52 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-08-06 23:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1387"
  %s.name         == "1API GmbH"
  %s.url          == "http://www.1api.net"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].email         == "https://whois.centralnic.com/contact/ntt.jp.net/registrant"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].email         == "https://whois.centralnic.com/contact/ntt.jp.net/admin"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].email         == "https://whois.centralnic.com/contact/ntt.jp.net/tech"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "f1g1ns1.dnspod.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "f1g1ns2.dnspod.net"
